images/win/scripts/Installers/Install-Edge.ps1

```diff
@@ -20,22 +20,15 @@ Write-Host "Get the Microsoft Edge WebDriver version..."
 $RegistryPath = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths"
 $EdgePath = (Get-ItemProperty "$RegistryPath\msedge.exe").'(default)'
 [version]$EdgeVersion = [System.Diagnostics.FileVersionInfo]::GetVersionInfo($EdgePath).ProductVersion
-$EdgeDriverVersionUrl = "https://msedgedriver.azureedge.net/LATEST_RELEASE_$($EdgeVersion.Major)"
-
-$EdgeDriverVersionFile = Start-DownloadWithRetry -Url $EdgeDriverVersionUrl -Name "versioninfo.txt" -DownloadPath $EdgeDriverPath
 
 Write-Host "Download Microsoft Edge WebDriver..."
-$EdgeDriverLatestVersion = Get-Content -Path $EdgeDriverVersionFile
+$EdgeDriverVersion = $EdgeVersion.ToString()
```

Comment:
Are you sure 100% sure that the Edge and Edge Driver are released at the same time (and even minute)?
In case, if Edge Driver is released in 3 hours later than Edge, image generation CI will be broken for those 3 hours.

The better approach will be using `LATEST_RELEASE` but for specific system.
For example, there is https://msedgewebdriverstorage.blob.core.windows.net/edgewebdriver/LATEST_RELEASE_89_WINDOWS

So we can use `$EdgeDriverVersionUrl = "https://msedgedriver.azureedge.net/LATEST_RELEASE_$($EdgeVersion.Major)_WINDOWS"`